<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>word_choose</class>
 <widget class="QWidget" name="word_choose">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>459</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>选择词库</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#word_choose{border-image: url(:/Yosemite-Color-Block.png);}</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(30, 84, 145);</string>
   </property>
   <property name="text">
    <string>选择词库:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="but_from_high">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>高中</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_from_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>四级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_from_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>六级</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_back">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>390</y>
     <width>141</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">/**正常情况下样式**/
QPushButton{
	font: 15pt &quot;微软雅黑&quot;;
    color: #1e5491;
    background-color: #eec0da;
    border-color: #567cbe;
    border-radius: 17px;
    border-style: solid;
    border-width: 3px;
    padding: 5px;
}

/**鼠标停留在按钮上的样式**/
QPushButton::hover{	
    color: #1e5491;
    background-color: #c0a1cc;
    border-color: #567cbe;
}

/**鼠标按压下去的样式**/
QPushButton::pressed,QPushButton::checked{
    color: #eddfed;
    background-color: #1c58a3;
}

/**按钮失能情况下样式**/
QPushButton::disabled{
    color: #FFFFFF;
    background-color: #dcdde1;
    border-color: #dcdde1;
}
</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
